ckcp/openshift_dev_setup.sh

```diff
@@ -419,6 +419,9 @@ main() {
   done
   echo [sync]
   register_compute
+  printf "\nUse the below KUBECONFIG to get access to the kcp workspace and compute cluster respectively.\n"
+  printf "KUBECONFIG_KCP: %s\n" "$KUBECONFIG_KCP"
+  printf "KUBECONFIG: %s\n" "$KUBECONFIG"
```

Comment:
Do you think there would be value in proposing the user to set some aliases? E.g.
```
  printf "\nYou can set the following aliases to access to the kcp workspace and compute cluster respectively.\n"
  printf "alias kkcp='KUBECONFIG=%s kubectl'" "$KUBECONFIG_KCP"
  printf "alias kcompute='KUBECONFIG=%s kubectl'" "$KUBECONFIG"
```

Reply:
Good idea. Since not everybody uses alias, I have kept both sets of printf statements.